bundles/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/ApiAccess.java

```diff
@@ -37,17 +39,17 @@
  * @author Jasper van Zuijlen - Initial contribution
  *
  */
+@NonNullByDefault
 public class ApiAccess {
     private static final int REQUEST_TIMEOUT_SECONDS = 5;
     private final Logger logger = LoggerFactory.getLogger(ApiAccess.class);
     private final HttpClient httpClient;
-    private final Gson gson;
+    private final @Nullable Gson gson = new GsonBuilder().create();
```

Comment:
Previously it didn't have to be nullable because constructor was guaranteed to assign it. What is the advantage of this change?

Reply:
Your right.